src/js/controllers/confirm.js

```diff
@@ -69,15 +69,15 @@ angular.module('copayApp.controllers').controller('confirmController', function(
 
   $scope.$on("$ionicView.beforeEnter", function(event, data) {
 
-    function setWalletSelector(coin, network, minAmount, cb) {
+    function setWalletSelector(coin, network, minAmount, sendMax, cb) {
```

Comment:
maybe better to refacto and send coin ==null  or send  the whole `tx`

Reply:
Good idea. Thanks